reg/drone/phy/thermodynamics/PressureTempCovTs.0.1.uavcan

```diff
@@ -0,0 +1,15 @@
+# Timestamped fluid pressure and temperature (sampled synchronously) with covariance.
+# Observe that this is a structural subtype of uavcan.si.sample.pressure.Scalar.1.0.
+
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+
+uavcan.si.unit.pressure.Scalar.1.0    pressure
+uavcan.si.unit.temperature.Scalar.1.0 temperature
+
+float16[3] covariance
```

Comment:
How does this define the URT? These should be `float16[6]` then.

Reply:
You are missing that the covariance matrix is 2x2 elements, so its URT contains 3 elements. This is correct.